frontend/packages/console-shared/src/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export function prefixedId(idPrefix, id) {
```

Comment:
```suggestion
export function prefixedID(idPrefix, id) {
```

I don't think this helper is really that helpful. Can't we use a string template directly?